repos/system_upgrade/el7toel8/actors/satelliteupgradecheck/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.satelliteupgradecheck import satelliteupgradecheck
+from leapp.models import Report, SatelliteFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class SatelliteUpgradeCheck(Actor):
+    """
+    Check state of Satellite system before upgrade
+    """
+
+    name = 'satelliteupgradecheck'
+    consumes = (SatelliteFacts,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    def process(self):
+        facts = next(self.consume(SatelliteFacts), None)
+        if not facts or not facts.has_foreman:
+            return
+
+        satelliteupgradecheck(facts)
```

Comment:
Same here, could you please rename the module to have underscores `satellite_upgrade_check`? Sorry I missed that in the first review.

Reply:
You didn't we refactored it today, that's why